Jellyfin.Api/Controllers/SyncPlayController.cs

```diff
@@ -92,31 +97,137 @@ public ActionResult SyncPlayLeaveGroup()
         /// <summary>
         /// Gets all SyncPlay groups.
         /// </summary>
-        /// <param name="filterItemId">Optional. Filter by item id.</param>
         /// <response code="200">Groups returned.</response>
         /// <returns>An <see cref="IEnumerable{GroupInfoView}"/> containing the available SyncPlay groups.</returns>
         [HttpGet("List")]
         [ProducesResponseType(StatusCodes.Status200OK)]
-        public ActionResult<IEnumerable<GroupInfoView>> SyncPlayGetGroups([FromQuery] Guid? filterItemId)
+        public ActionResult<IEnumerable<GroupInfoDto>> SyncPlayGetGroups()
         {
             var currentSession = RequestHelpers.GetSession(_sessionManager, _authorizationContext, Request);
-            return Ok(_syncPlayManager.ListGroups(currentSession, filterItemId.HasValue ? filterItemId.Value : Guid.Empty));
+            return Ok(_syncPlayManager.ListGroups(currentSession));
         }
 
         /// <summary>
         /// Request play in SyncPlay group.
         /// </summary>
+        /// <param name="playingQueue">The playing queue. Item ids in the playing queue, comma delimited.</param>
+        /// <param name="playingItemPosition">The playing item position from the queue.</param>
+        /// <param name="startPositionTicks">The start position ticks.</param>
         /// <response code="204">Play request sent to all group members.</response>
         /// <returns>A <see cref="NoContentResult"/> indicating success.</returns>
         [HttpPost("Play")]
         [ProducesResponseType(StatusCodes.Status204NoContent)]
-        public ActionResult SyncPlayPlay()
+        public ActionResult SyncPlayPlay(
+            [FromQuery, Required] string playingQueue,
```

Comment:
```suggestion
            [FromQuery, Required] Guid[] playingQueue,
```

Please avoid using the `RequestHelpers` to parse parameters on new endpoints.